### ISSUE TITLE
[URGENT] Library access issue

### ISSUE THREAD

**Issue by tananaev** [cdnjs/cdnjs#11212]
I am trying to access extjs library on cdnjs:

http://cdnjs.cloudflare.com/ajax/libs/extjs/6.2.0/ext-all-debug.js

...but there seems to be some issue with the server:

![screen shot 2017-05-10 at 2 40 13 pm](https://cloud.githubusercontent.com/assets/1161863/25881022/ce1406e2-358e-11e7-91df-c35bc1dff427.png)

**Comment by hare1039** [comment]
@PeterDaveHello Please take a look at this issue.
@tananaev Hi, Please wait for main maintainer of cdnjs to see what's going on. Thanks for reporting.

**Comment by hare1039** [comment]
BTW, I can access this library in Taiwan currently, would you please see if it is working now?

**Comment by tananaev** [comment]
I am located in New Zealand. Request seems to be going though Sydney server, as you can see from screenshot.

**Comment by iArren** [comment]
I am located at Russia and cdnjs resources (specifically JS) is not available for .loc and .dev domains, it just drops connection and gives 522.

**Comment by nuclearspike** [comment]
I'm having CORS issues with cloudflare that just started. Don't know if this is related but it just started tonight.
![screen shot 2017-05-09 at 10 33 44 pm](https://cloud.githubusercontent.com/assets/1151638/25883409/b0764974-3507-11e7-8606-92c1bf4499ec.png)

**Comment by nuclearspike** [comment]
BTW: That was also happening in production, I switched to another CDN (npmcdn.com) because those files not loading broke my site.

**Comment by damnedhorse** [comment]
I'm seeing it in London.  Resources blocked by CORS.

**Comment by eifoshm** [comment]
Same issue with
https://cdnjs.cloudflare.com/ajax/libs/react-slick/0.14.8/react-slick.min.js
![Uploading Screenshot_2017-05-10-10-33-00_com.android.chrome.png…]()

**Comment by terinjokes** [comment]
We're looking into why the origin server is timing out some requests.

**Comment by terinjokes** [comment]
We've resolved the issue at this time.

We're unsure why the box handling uncached requests suddenly died, or why alerts never fired for it. We'll investigate why and make the changes necessary.

**Comment by pkra** [comment]
@terinjokes I'm now getting a 403 (in Germany) at https://cdnjs.cloudflare.com/ajax/libs/mathjax/2.7.1/extensions/a11y/mathmaps/symbols/math_arrows.json. Any thoughts?

**Comment by pkra** [comment]
Ok, json-related problems are a separate issue. See the discussion at https://github.com/cdnjs/cdnjs/issues/10160#issuecomment-270032154.

**Comment by DovydasNavickas** [comment]
Still getting 522 in Lithuania for React here:
https://cdnjs.cloudflare.com/ajax/libs/react/15.4.2/react-dom.min.js

**Comment by DovydasNavickas** [comment]
VM in the Netherlands loads everything perfectly though... 😕

**Comment by Maikuolan** [comment]
Currently replying from Perth, Western Australia.

Attempting to access the resource in question, I see:

```
/*
This file is part of Ext JS 6.2.0.981

Copyright (c) 2011-2016 Sencha Inc

Contact:  http://www.sencha.com/contact

GNU General Public License Usage
This file may be.. (and so on)
```

So, seems to be working on my end, I guess.
